Semver/Ranges/Comparers/Npm/NpmParseOptions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+
+namespace Semver.Ranges.Comparers.Npm
+{
+    /// <summary>
+    /// The options to use when parsing a range with npm syntax.
+    /// </summary>
+    public readonly struct NpmParseOptions : IEquatable<NpmParseOptions>
```

Comment:
If there is a possibility the set of options to be expanded in future versions, then this should be a `class`. If it is unlikely the options should be expanded, then I would eliminate this type altogether and just make it a boolean parameter to the parse and satisfies methods.

Reply:
The original implementation also has a ```loose``` option. There is potential for that being implemented if someone requests it but i'm not personally inclined to at this time.

I'd say it's more future proof to keep it as a struct or class rather than a bool value, but it's up to you.